Felienne/hedy
# Create lark dynamically

**Description**
In this PR we enable the dynamic creating of the .lark grammar file for the different languages. Currently we perform this action partly manually and partly automated. We create a template file with placeholders which we overwrite with translation retrieved from the YAML keyword files. We also take into account the edge case where translations might have multiple options.

**Fixes**
This PR fixes #2726 and #2682

**How to test**
Run the `transform_keywords_to_grammar.py` file in the `/content` folder. All .lark files should automatically be created. Notice that they are already created in this PR so running the file is not necessary. Other than that, if the tests pass this is ready to go!
